cmd/hubagent/options/validation.go

```diff
@@ -37,10 +37,6 @@ func (o *Options) Validate() field.ErrorList {
 		errs = append(errs, field.Invalid(newPath.Child("WorkPendingGracePeriod"), o.WorkPendingGracePeriod, "Must be greater than 0"))
 	}
 
-	if o.EnableWebhook && o.WebhookServiceName == "" {
-		errs = append(errs, field.Invalid(newPath.Child("WebhookServiceName"), o.WebhookServiceName, "Webhook service name is required when webhook is enabled"))
-	}
-
 	connectionType := o.WebhookClientConnectionType
```

Comment:
you cannot remove this check. eg, user sets the webhookServiceName explicitly. In this case, we need to return the error.